src/packages/solid/components/Metadata/Details.tsx

```diff
@@ -39,7 +39,12 @@ export interface DetailsProps extends IntrinsicNodeProps {
 const Details: Component<DetailsProps> = (props: DetailsProps) => {
   return (
     <View
-      style={styles.Container}
+      style={[
+        props.style?.Container,
+        props.style?.Container?.[props.tone || styles.tone],
```

Comment:
no . before [

Reply:
with out it there were errors with it thinking its a ternary